### PR TITLE
[TASK] add username scope to device token

### PR DESCRIPTION
**Pull request by shekenahglory**

### CODE REVIEW

File: src/js/services/id.js [hunk 1]
```diff
@@ -225,6 +225,21 @@ module.factory('rpId', ['$rootScope', '$location', '$route', '$routeParams', '$t
     store.set('ripple_auth', {url:url, username: username, keys: keys});
   };
 
+  Id.prototype.setDeviceID = function (username, device_id) {
+    Id.normalizeUsernameForInternals(username);
+    return store.set(username + '|device_id', device_id);  
+  };
```
Comment (MatthewPhinney) [inline src/js/services/id.js]: Are you setting the normalized username, or the passed in username on the store?

Comment (vhpoet) [inline src/js/services/id.js]: @shekenahglory

Comment (shekenahglory) [inline src/js/services/id.js]: haha oops.  I intended to normalize it.

**Comment by vhpoet** [comment]
LGTM

**Pull request merged**